### PR TITLE
Issue 8592: Detailed message for missing config

### PR DESCRIPTION
**Pull request by perillai**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

This change is related to [Issue 8592](https://github.com/webdriverio/webdriverio/issues/8592). Link to the missing config file is provided for more clarity to the user while using run.

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### Reviewers: @webdriverio/project-committers

<a href="https://gitpod.io/#https://github.com/webdriverio/webdriverio/pull/8630"></a>

### CODE REVIEW

File: packages/wdio-cli/src/commands/config.ts [hunk 1]
```diff
@@ -255,11 +255,17 @@ export function handler(argv: ConfigCommandArguments) {
  * @param {Function} runConfigCmd   runConfig method to be replaceable for unit testing
  */
 export async function missingConfigurationPrompt(command: string, message: string, useYarn = false, runConfigCmd = runConfig) {
+    let configMessage: string
+    if (command === 'run'){
+        configMessage = `Error: Could not execute "run" due to missing configuration, file "${message}" not found! Would you like to create one?`
+    } else {
+        configMessage = `Error: Could not execute "${command}" due to missing configuration. Would you like to create one?`
+    }
```
Comment (christian-bromann) [inline packages/wdio-cli/src/commands/config.ts:263]:
```suggestion
    const configMessage = command === 'run'
        ? `Error: Could not execute "run" due to missing configuration, file "${message}" not found! Would you like to create one?`
        : `Error: Could not execute "${command}" due to missing configuration. Would you like to create one?`
```

File: packages/wdio-cli/src/commands/config.ts [hunk 2]
```diff
@@ -269,7 +275,11 @@ export async function missingConfigurationPrompt(command: string, message: strin
      */
     if (!config && !process.env.VITEST_WORKER_ID) {
         /* istanbul ignore next */
-        console.log(message)
+        if (command === 'run'){
+            console.log(`No WebdriverIO configuration found in "${process.cwd()}"`)
+        } else {
+            console.log(message)
+        }
```
Comment (christian-bromann) [inline packages/wdio-cli/src/commands/config.ts:282]:
```suggestion
        console.log(command === 'run'
            ? `No WebdriverIO configuration found in "${process.cwd()}"`
            : message
```

**Review by christian-bromann — COMMENTED** [review]
Minor code style improvements. Also it seems some tests are failing.

File: packages/wdio-cli/src/commands/config.ts (hunk 2 above)
Comment (perillai) [reply packages/wdio-cli/src/commands/config.ts]: @christian-bromann Thanks for the suggestions. Committed with the changes you had suggested and also the tests are passing in local now.

**Review by christian-bromann — APPROVED** [review]
👍 @perillai thank you

**Comment by christian-bromann** [comment]
We currently develop a new major WebdriverIO version `v8` on the `main` branch. Mind raising the same PR on the `v7` branch? I am happy to do this too but not sure when I get to it.

**Comment by perillai** [comment]
> This change is related to [Issue 8592](https://github.com/webdriverio/webdriverio/issues/8592). Link to the missing config file is provided for more clarity to the user while using run.

Have raised PR with the same code to the v7 branch [here](https://github.com/webdriverio/webdriverio/pull/8633).

**Pull request merged**